src/Quizzer.API.DAL/Seeds/Storage.cs

```diff
@@ -0,0 +1,559 @@
+            new Guid("8CC8198A-A4E0-47CD-A1C1-35DF84FB60EE"),
+            new Guid("8CC8198A-A4E0-47CD-A1C1-35DF84FB60EF"),
+            new Guid("F390B866-8251-4CB8-B4A5-B97D39B85F40"),
+            new Guid("F390B866-8251-4CB8-B4A5-B97D39B85F41"),
+            new Guid("F390B866-8251-4CB8-B4A5-B97D39B85F42"),
+            new Guid("F390B866-8251-4CB8-B4A5-B97D39B85F43"),
+            new Guid("F390B866-8251-4CB8-B4A5-B97D39B85F44")
+        };
+        private IList<Guid> SelectedAnswerGuids { get; } = new List<Guid>() {
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC50"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC51"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC52"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC53"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC54"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC55"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC56"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC57"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC58"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC59"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC5A"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC5B"),
+            new Guid("BE3D1D84-AE2F-4996-A6D8-5B652383CC5C"),
+        };
+        private IList<Guid> UserGuids { get; } = new List<Guid>() {
+            new Guid("9B68BC92-DF04-48EE-A7A4-5060E517DFCC"),
+            new Guid("9B68BC92-DF04-48EE-A7A4-5060E517DFCD")
+        };
+        private IList<Guid> UserAnswerGuids { get; } = new List<Guid>() {
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA70"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA71"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA72"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA73"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA74"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA75"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA76"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA77"),
+            new Guid("A6E5C185-71F0-42AE-A9E5-4AC47AACCA78")
+        };
+
+        public Storage(bool seedData) {
+            if (seedData) {
+                SeedQuizzes();
+                SeedUsers();
+                SeedQuestions();
+                SeedAnswers();
+                SeedCompletedQuizzes();
+                SeedUserAnswers();
+                SeedSelectedAnswers();
+            }
+        }
+
+        private void SeedQuizzes() {
+            Quizzes.Add(new QuizEntity {
+                Id = QuizGuids[0],
+                StartTime = DateTime.Parse("15.01.2023 00:00", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                EndTime = DateTime.Parse("15.01.2024 19:34", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
```

Comment:
Why there is french culture?

Reply:
I like this format and i wanted to be sure, that those datetimes will be parsed correctly.

Reply:
cs-cz does not have this format? I don't mind it that at all, but it surprised me choosing this culture.